website/docs/teleop/vr.md

```diff
@@ -3,26 +3,30 @@ title: VR
 sidebar_position: 2
 ---
 
-# OpenArm VR Teleoperation Overview
-This project enables real-time teleoperation of the OpenArm robot using VR human body estimation streaming data. 
+# VR
+
+## OpenArm VR Teleoperation Overview
+
+This project enables real-time teleoperation of the OpenArm robot using VR human body estimation streaming data.
 
 Coming soon!
 # Key Features
```

Comment:
> # Key Features

Need to fix this too?

Reply:
fix: [e8be283](https://github.com/enactic/openarm/pull/177/commits/e8be28330fc1a85ec1ec5725d7bcefef515e6233)